zcash_client_sqlite/src/wallet.rs

```diff
@@ -1232,26 +1231,21 @@ pub fn put_sent_utxo<'a, P: consensus::Parameters>(
 ///   transaction.
 /// - If `to` is a transparent address, this is an index into the transparent outputs of
 ///   the transaction.
-#[deprecated(
-    note = "This method will be removed in a future update. Use zcash_client_backend::data_api::WalletWrite::store_sent_tx instead."
-)]
-pub fn insert_sent_note<'a, P: consensus::Parameters>(
+pub(crate) fn insert_sent_note<'a, P: consensus::Parameters>(
     stmts: &mut DataConnStmtCache<'a, P>,
     tx_ref: i64,
     output_index: usize,
-    account: AccountId,
-    to: &PaymentAddress,
+    from_account: AccountId,
+    to: &Recipient,
```

Comment:
The type here is invalid, as it allows a transparent address to be passed in.

Reply:
It's actually not; see https://github.com/zcash/librustzcash/pull/659/commits/179e22b76d8a34eff7cfc995683dcf34692816ac

Reply:
It _was_ invalid at the time (it's not possible to have a transparent recipient in the Sapling pool), but it looks like the most recent commit fixes it.